### PR TITLE
Feature/iii 3227

### PR DESCRIPTION
**Pull request by brampauwelyn**
### Added
- added delete address endpoint to udb-api service
- added delete endpoint to organizer manager service
- added button + ui messages for address deletion in organizer edit html template

---
Ticket: [https://jira.uitdatabank.be/browse/III-3227](https://jira.uitdatabank.be/browse/III-3227)

### CODE REVIEW

File: src/core/translations/dutch-translations.constant.js [hunk 1]
```diff
@@ -841,7 +841,10 @@ angular.module('udb.core')
         'help_city': 'Er was een probleem tijdens het ophalen van de steden.',
         'error_city': 'Gelieve een gemeente in te geven.',
         'change': 'Wijzigen',
-        'zip': 'Postcode'
+        'zip': 'Postcode',
+        'delete': 'Leegmaken',
+        'delete_success': 'Adres succesvol verwijderd',
+        'delete_error': 'Er liep iets fout tijdens het verwijderen van jouw address'
```
Comment (jdrieghe) [inline src/core/translations/dutch-translations.constant.js:847]:
```suggestion
        'delete_error': 'Er liep iets fout tijdens het verwijderen van jouw adres'
```

File: src/organizers/components/organizer-address/organizer-address.component.spec.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ describe('Component: Organizer Address', function() {
     streetAddress: 'Sluisstraat 79'
   };
 
+
```
Comment (jdrieghe) [inline src/organizers/components/organizer-address/organizer-address.component.spec.js:13]:
```suggestion
```

File: src/organizers/components/organizer-address/organizer-address.component.spec.js [hunk 3]
```diff
@@ -34,7 +44,8 @@ describe('Component: Organizer Address', function() {
     $componentController = _$componentController_;
     citiesBE = $injector.get('citiesBE');
     citiesNL = $injector.get('citiesNL');
-    Levenshtein = $injector.get('Levenshtein')
+    Levenshtein = $injector.get('Levenshtein');
+
```
Comment (jdrieghe) [inline src/organizers/components/organizer-address/organizer-address.component.spec.js:48]:
```suggestion
```

File: src/core/translations/dutch-translations.constant.js (hunk 1 above)
Comment (brampauwelyn) [reply src/core/translations/dutch-translations.constant.js]: @jdrieghe nice catch ;). I also forgot to add the FR translations so I added them now as well.

**Pull request merged**